src/App.css

```diff
@@ -0,0 +1,5 @@
+#square {
```

Comment:
square I imagine should be a class since there will be multiple squares on a page. Update to be `.square`